spec/services/invoices/payments/payment_providers/factory_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Invoices::Payments::PaymentProviders::Factory, type: :service do
+  subject(:factory_service) { described_class.new_instance(invoice:) }
+
+  let(:payment_provider) { 'stripe' }
+  let(:invoice) { create(:invoice, customer:) }
+  let(:customer) { create(:customer, payment_provider:) }
+
+  describe '#self.new_instance' do
+    context 'when stripe' do
+      it  'returns correct class' do
```

Comment:
Layout/SpaceBeforeFirstArg: Put one space between the method name and the first argument.